### PR TITLE
Feat/init

### PR DESCRIPTION
**Pull request by yannickglt**

### CODE REVIEW

**Comment by firehist** [comment]
Lol @yannickglt is this project handle real-time analytics over a 3D worldmap?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "angular-cli": {},
+  "scripts": {
+    "start": "ng serve",
+    "lint": "tslint \"src/**/*.ts\"",
+    "test": "ng test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
+  },
+  "dependencies": {
+    "@angular/common": "2.4.1",
```
Comment (firehist) [inline package.json:14]: @yannickglt Be a bit more optimistic ...
https://github.com/angular/angular/releases/tag/4.0.0-beta.1

File: package.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "@angular/compiler": "2.4.1",
+    "@angular/core": "2.4.1",
+    "@angular/forms": "2.4.1",
+    "@angular/http": "2.4.1",
+    "@angular/platform-browser": "2.4.1",
+    "@angular/platform-browser-dynamic": "2.4.1",
+    "@angular/router": "3.2.3",
+    "core-js": "^2.4.1",
+    "moment": "^2.17.1",
+    "rxjs": "5.0.0-beta.12",
```
Comment (firehist) [inline package.json:24]: https://github.com/ReactiveX/rxjs/releases/tag/5.0.2 🦄

**Review by firehist — COMMENTED** [review]
Lovely PR :D

**Comment by yannickglt** [comment]
> Lol @yannickglt is this project handle real-time analytics over a 3D worldmap?

Ahaa it could be 😄

File: package.json (hunk 1 above)
Comment (yannickglt) [reply package.json:14]: I just used the angular-cli for now. But yeah, it should be cool!

**Comment by firehist** [comment]
Awesome :) 🦄 🚀 💃

**Pull request merged**